sdk/communication/communication-sms/src/generated/src/smsApiClientContext.ts

```diff
@@ -10,7 +10,7 @@ import * as coreHttp from "@azure/core-http";
 import { SmsApiClientOptionalParams } from "./models";
 
 const packageName = "azure-communication-sms";
-const packageVersion = "1.0.0-beta.4";
+const packageVersion = "1.0.0";
```

Comment:
How did that happen? Should be "1.0.0-beta.5"

Reply:
@DominikMe It looks generated from sms. Should I make a manual change to change it to `beta.5` ?